index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style/style.css" />
+  </head>
+  <body id="main">
+    <title id="title">Dr. Norman Borlaug</title>
+    <div id="img-div">
+      <h1>Dr. Norman Borlaugs</h1>
+      <img class="responsive" src="/img/Dr Norman.jpg" alt="Norman Borlaug" />
+      <caption id="img-caption">
+        About Dr. Norman Borlaug:
```

Comment:
I think this About Dr. Norman Borlaugh would look better below the image.

Reply:
You did a great job. 👏 
- [ ] OPTIONAL
- Kindly use sentences like "Please, I think this About Dr. Norman Borlaugh would look better below the image".
- Please, always includes an explanation, a "why", you requested change. Example: "Please, I think this About Dr. Norman Borlaugh would look better below the image, because ......."